spec/features/rooms/favourite_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.describe "Favourite rooms", type: :feature do
+  before do
+    @user = create(:user)
+    sign_in @user
+    @room = create(:room)
+  end
+
+  it "Favourite button adds room to favourite", js: true do
+    visit room_path(@room)
+    page.click("room-favourite")
+    visit room_path(@room)
+    expect(page.find("#room-favourite")).to have_css("selected")
```

Comment:
have_css doesn't work this way. You have to expect a parent (like `expect(page).to have_css`). But no idea how to ensure that it is actually the button that is selected

Reply:
fixed